### DISCUSSION TITLE
تعديل

### DISCUSSION THREAD

**Discussion by tachi455** [thatskymod/Sky-CotL-Scripts#104] [General]
نطالب ب تحديث او تعديل Dukun Langit  لانه لم يعد يعمل بشكل صحيح و ايضا Rome

**Comment by thatskymod** [comment]
With Dukun Langit now updated to [0.4.5](https://github.com/thatskymod/Sky-CotL-Scripts/blob/6a5d6946eb5661e5e14f714d56ca0f4157b3c351/Canvas/Libs/libLangit045.so), there should be no more issues in running it.

LibRomanBeta isn't supported at the moment, but there’s no harm in keeping it installed. This way, you'll automatically receive an update as soon as it becomes available.
